Documentation/LanguageServer/colorization.md

```diff
@@ -0,0 +1,87 @@
+
+# VS Code C/C++ Extension - Enhanced Colorization
+
+The VS Code C/C++ extension now supports lexical/syntactic and semantic colorization, using IntelliSense.
+
```

Comment:
Is this comma supposed to be here? Should you explicitly say whether the lexical colorization requires IntelliSense or not?

Reply:
I think the comma is as intended, as the 'using' applies to more than just 'semantic colorization'.  I was being intentionally vague for simplicity, rather than explain the difference between lexical/syntactic and semantic approaches, as that distinction may not be meaningful to most people.  If IntelliSense is disabled, the intention was that enhancedColorization would be fully disabled, rather than provide only lexical/syntactic.  The setting documentation includes: "This setting has no effect if IntelliSense is disabled."  … That said, it looks like I'll need to address some issues with enabling/disabling IntelliSense, to ensure that enables/disables colorization properly.

Reply:
Maybe say something like "The extensions now supports feature B when feature A is enabled." or with the use of IntelliSense.